### PR TITLE
Add results links and text for all questions that have them

### PR DESCRIPTION
**Pull request by issyl0**
- We've not included the new DEFRA question.
- The only question missing results on the spreadsheet is the final one:
  "Are you able to leave your home if absolutely necessary?"

Co-authored-by: Issy Long <issy.long@digital.cabinet-office.gov.uk>
Co-authored-by: Jon Hallam <jonathan.hallam@digital.cabinet-office.gov.uk>

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -241,3 +241,218 @@ en:
...
+          href: "https://phw.nhs.wales/topics/latest-information-on-novel-coronavirus-covid-19/staying-well-at-home/how-are-you-feeling/"
+        - text: "Get information about coronavirus and your mental wellbeing from the Scottish Association for Mental Health"
+          href: "https://www.samh.org.uk/about-mental-health/self-help-and-wellbeing/coronavirus-and-your-mental-wellbeing"
+        - text: "Find your local loneliness service"
+          href: "https://www.redcross.org.uk/get-help/get-help-with-loneliness/find-your-local-loneliness-service"
+        - text: "If you need urgent help now, call the Samaritans on 116 123"
+        - text: "If you need urgent help text ‘Shout’ to 85258"
+        - text: "If you need urgent help and you’re in Northern Ireland call Lifeline on 0808 808 8000"
+        - text: "If you need urgent help and you’re in Wales call 0800 132 737 or text ‘help’ to 81066"
+        - text: "If it’s an emergency, call 999"
```
Comment (alex-ju) [inline config/locales/en.yml:458]:
```suggestion
        - text: "If it’s an emergency, call <a href="tel:999" class="govuk-link">999</a>"
```

**Comment by issyl0** [comment]
Thanks Alex, that's great. I'm going to merge this so it's not blocking anything else (and so that I don't have to rebase when the error pages work starts) - we can add more things later when we have further answers (for the last question, hopefully? it's weird that it doesn't have any).

File: config/locales/en.yml [hunk 2]
```diff
@@ -241,3 +241,218 @@ en:
...
+        - text: "Get information about taking care of your mental health if you live in Northern Ireland"
+          href: "https://www.nidirect.gov.uk/articles/coronavirus-covid-19-taking-care-your-mental-health-and-wellbeing"
+        - text: "Get information about taking care of your mental health if you live in Wales"
+          href: "https://phw.nhs.wales/topics/latest-information-on-novel-coronavirus-covid-19/staying-well-at-home/how-are-you-feeling/"
+        - text: "Get information about coronavirus and your mental wellbeing from the Scottish Association for Mental Health"
+          href: "https://www.samh.org.uk/about-mental-health/self-help-and-wellbeing/coronavirus-and-your-mental-wellbeing"
+        - text: "Find your local loneliness service"
+          href: "https://www.redcross.org.uk/get-help/get-help-with-loneliness/find-your-local-loneliness-service"
+        - text: "If you need urgent help now, call the Samaritans on <a href="tel:03448 920 908" class="govuk-link">116 123</a>116 123</a>"
```
Comment (issyl0) [inline config/locales/en.yml:454]:
```suggestion
        - text: 'If you need urgent help now, call the Samaritans on <a href="tel:03448 920 908" class="govuk-link">116 123</a>116 123</a>'
```

File: config/locales/en.yml [hunk 3]
```diff
@@ -241,3 +241,218 @@ en:
...
+        - text: "If you need urgent help text ‘Shout’ to 85258"
+        - text: "If you need urgent help and you’re in Northern Ireland call Lifeline on <a href="tel:0808 808 8000" class="govuk-link">0808 808 8000</a>"
```
Comment (issyl0) [inline config/locales/en.yml:456]:
```suggestion
        - text: 'If you need urgent help and you’re in Northern Ireland call Lifeline on <a href="tel:0808 808 8000" class="govuk-link">0808 808 8000</a>'
```

File: config/locales/en.yml [hunk 4]
```diff
@@ -241,3 +241,218 @@ en:
...
+        - text: "If you need urgent help and you’re in Wales call <a href="tel:0800 132 737" class="govuk-link">0800 132 737</a> or text ‘help’ to 81066"
```
Comment (issyl0) [inline config/locales/en.yml:457]:
```suggestion
        - text: 'If you need urgent help and you’re in Wales call <a href="tel:0800 132 737" class="govuk-link">0800 132 737</a> or text ‘help’ to 81066'
```

File: config/locales/en.yml [hunk 5]
```diff
@@ -241,3 +241,218 @@ en:
...
+      show_options:
+        - "Yes"
+        - "I might be evicted soon"
+        - "Not sure"
+      links:
+        - text: "Get information about your rights as a tenant"
+          href: "https://www.gov.uk/government/publications/covid-19-and-renting-guidance-for-landlords-tenants-and-local-authorities"
+        - text: "Get coronavirus housing advice from Shelter"
+          href: "https://england.shelter.org.uk/housing_advice/coronavirus"
+        - text: "If you’re in Northern Ireland call the Housing Executive on <a href="tel:03448 920 908" class="govuk-link">03448 920 908</a>"
```
Comment (issyl0) [inline config/locales/en.yml:420]:
```suggestion
        - text: 'If you’re in Northern Ireland call the Housing Executive on <a href="tel:03448 920 908" class="govuk-link">03448 920 908</a>'
```

File: config/locales/en.yml [hunk 6]
```diff
@@ -241,3 +241,218 @@ en:
...
+          href: "https://www.victimsupport.org.uk/help-and-support/get-help/support-near-you/wales"
+    do_you_have_somewhere_to_live:
+      show_options:
+        - "I do now but I might lose it"
+        - "No"
+        - "Not sure"
+      links:
+        - text: "Get information if you are legally homeless from Shelter"
+          href: "https://england.shelter.org.uk/housing_advice/homelessness/rules/legally_homeless"
+        - text: "If you’re in Northern Ireland call the Housing Executive on <a href="tel:03448 920 908" class="govuk-link">03448 920 908</a>"
```
Comment (issyl0) [inline config/locales/en.yml:401]:
```suggestion
        - text: 'If you’re in Northern Ireland call the Housing Executive on <a href="tel:03448 920 908" class="govuk-link">03448 920 908</a>'
```

File: config/locales/en.yml [hunk 7]
```diff
@@ -241,3 +241,218 @@ en:
...
+    feel_safe:
+      show_options:
+        - "Yes but I’m concerned about the safety of someone else"
+        - "No"
+        - "Not sure"
+      links:
+        - text: "If you’re in immediate danger call <a href="tel:999" class="govuk-link">999</a> and ask for the Police."
+        - text: "If you’re in danger and unable to talk on the phone, call <a href="tel:999" class="govuk-link">999</a>, and then press 55."
+        - text: "Call the National Domestic Abuse Helpline on 0808 2000 247"
+        - text: "If you’re a child call Childline on <a href="tel:0800 1111" class="govuk-link">0800 1111</a>"
```
Comment (issyl0) [inline config/locales/en.yml:371]:
```suggestion
        - text: 'If you’re a child call Childline on <a href="tel:0800 1111" class="govuk-link">0800 1111</a>'
```

File: config/locales/en.yml [hunk 8]
```diff
@@ -241,3 +241,218 @@ en:
...
+        - text: 'Find helplines <a href="https://www.gov.uk/government/publications/coronavirus-covid-19-and-domestic-abuse/coronavirus-covid-19-support-for-victims-of-domestic-abuse" class="govuk-link">if you’re a victim of domestic abuse or feel at risk of abuse</a>, or if you are a <a href="https://www.gov.uk/guidance/coronavirus-covid-19-victim-and-witness-services" class="govuk-link">victim or witness of a crime</a> (these pages have no quick escape)'
+        - text: "Get safety advice for survivors from Woman's Aid"
+          href: "https://www.womensaid.org.uk/covid-19-coronavirus-safety-advice-for-survivors/"
+        - text: "Get information from Nexus NI if you're in Northern Ireland"
```
Comment (issyl0) [inline config/locales/en.yml:375]:
```suggestion
        - text: "Get information from Nexus NI if you’re in Northern Ireland"
```

File: config/locales/en.yml [hunk 9]
```diff
@@ -241,3 +241,218 @@ en:
...
+          href: "https://www.mygov.scot/school-meals/"
+        - text: "If you are in Northern Ireland call the coronavirus Community Helpline on <a href="tel:0808 802 0020" class="govuk-link">0808 802 0020</a>, email covid19@adviceni.net or text ‘ACTION’ to 81025"
+    feel_safe:
+      show_options:
+        - "Yes but I’m concerned about the safety of someone else"
+        - "No"
+        - "Not sure"
+      links:
+        - text: "If you’re in immediate danger call <a href="tel:999" class="govuk-link">999</a> and ask for the Police."
+        - text: "If you’re in danger and unable to talk on the phone, call <a href="tel:999" class="govuk-link">999</a>, and then press 55."
```
Comment (issyl0) [inline config/locales/en.yml:369]:
```suggestion
        - text: 'If you’re in danger and unable to talk on the phone, call <a href="tel:999" class="govuk-link">999</a>, and then press 55.'
```

File: config/locales/en.yml [hunk 10]
```diff
@@ -241,3 +241,218 @@ en:
...
+        - text: "Get information about getting free school meals during coronavirus if you live in Scotland"
+          href: "https://www.mygov.scot/school-meals/"
+        - text: "If you are in Northern Ireland call the coronavirus Community Helpline on <a href="tel:0808 802 0020" class="govuk-link">0808 802 0020</a>, email covid19@adviceni.net or text ‘ACTION’ to 81025"
+    feel_safe:
+      show_options:
+        - "Yes but I’m concerned about the safety of someone else"
+        - "No"
+        - "Not sure"
+      links:
+        - text: "If you’re in immediate danger call <a href="tel:999" class="govuk-link">999</a> and ask for the Police."
```
Comment (issyl0) [inline config/locales/en.yml:368]:
```suggestion
        - text: 'If you’re in immediate danger call <a href="tel:999" class="govuk-link">999</a> and ask for the Police.'
```

File: config/locales/en.yml [hunk 11]
```diff
@@ -241,3 +241,218 @@ en:
...
+          href: "https://www.gov.uk/universal-credit/eligibility"
+        - text: "If you are at least 10 weeks pregnant or have a child under 4, find out if you can apply for Healthy Start vouchers"
+          href: "https://www.healthystart.nhs.uk/healthy-start-vouchers/do-i-qualify/"
+        - text: "Find out if you're eligible for the Scottish Welfare Fund if you live in Scotland"
+          href: "https://www.mygov.scot/scottish-welfare-fund/"
+        - text: "Find out if you can get a grant if you're a parent or looking after a child in Scotland"
+          href: "https://www.mygov.scot/best-start-grant-best-start-foods/"
+        - text: "Get information about getting free school meals during coronavirus if you live in Scotland"
+          href: "https://www.mygov.scot/school-meals/"
+        - text: "If you are in Northern Ireland call the coronavirus Community Helpline on <a href="tel:0808 802 0020" class="govuk-link">0808 802 0020</a>, email covid19@adviceni.net or text ‘ACTION’ to 81025"
```
Comment (issyl0) [inline config/locales/en.yml:361]:
```suggestion
        - text: 'If you are in Northern Ireland call the coronavirus Community Helpline on <a href="tel:0808 802 0020" class="govuk-link">0808 802 0020</a>, email covid19@adviceni.net or text ‘ACTION’ to 81025'
```

File: config/locales/en.yml [hunk 12]
```diff
@@ -241,3 +241,218 @@ en:
...
+    feel_safe:
+      show_options:
+        - "Yes but I’m concerned about the safety of someone else"
+        - "No"
+        - "Not sure"
+      links:
+        - text: "If you’re in immediate danger call <a href="tel:999" class="govuk-link">999</a> and ask for the Police."
+        - text: "If you’re in danger and unable to talk on the phone, call <a href="tel:999" class="govuk-link">999</a>, and then press 55."
+        - text: "Call the National Domestic Abuse Helpline on 0808 2000 247"
```
Comment (issyl0) [inline config/locales/en.yml:370]:
```suggestion
        - text: 'Call the National Domestic Abuse Helpline on <a href="tel:0808 2000 247" class="govuk-link">0808 2000 247</a>'
```

File: config/locales/en.yml [hunk 13]
```diff
@@ -241,3 +241,218 @@ en:
...
+          href: "https://www.gov.uk/universal-credit/eligibility"
+        - text: "If you are at least 10 weeks pregnant or have a child under 4, find out if you can apply for Healthy Start vouchers"
+          href: "https://www.healthystart.nhs.uk/healthy-start-vouchers/do-i-qualify/"
+        - text: "Find out if you're eligible for the Scottish Welfare Fund if you live in Scotland"
+          href: "https://www.mygov.scot/scottish-welfare-fund/"
+        - text: "Find out if you can get a grant if you're a parent or looking after a child in Scotland"
+          href: "https://www.mygov.scot/best-start-grant-best-start-foods/"
+        - text: "Get information about getting free school meals during coronavirus if you live in Scotland"
+          href: "https://www.mygov.scot/school-meals/"
+        - text: 'If you are in Northern Ireland call the coronavirus Community Helpline on <a href="tel:0808 802 0020" class="govuk-link">0808 802 0020</a>, email covid19@adviceni.net or text ‘ACTION’ to 81025'
```
Comment (injms) [inline config/locales/en.yml:361]: `tel:` links need to include the country code:

```suggestion
        - text: 'If you are in Northern Ireland call the coronavirus Community Helpline on <a href="tel:+448088020020" class="govuk-link">0808 802 0020</a>, email covid19@adviceni.net or text ‘ACTION’ to 81025'
```

File: config/locales/en.yml [hunk 14]
```diff
@@ -241,3 +241,218 @@ en:
...
+          href: "https://www.victimsupport.org.uk/help-and-support/get-help/support-near-you/wales"
+    do_you_have_somewhere_to_live:
+      show_options:
+        - "I do now but I might lose it"
+        - "No"
+        - "Not sure"
+      links:
+        - text: "Get information if you are legally homeless from Shelter"
+          href: "https://england.shelter.org.uk/housing_advice/homelessness/rules/legally_homeless"
+        - text: 'If you’re in Northern Ireland call the Housing Executive on <a href="tel:03448 920 908" class="govuk-link">03448 920 908</a>'
```
Comment (injms) [inline config/locales/en.yml:401]: Same as previous:

```suggestion
        - text: 'If you’re in Northern Ireland call the Housing Executive on <a href="tel:+443448920908" class="govuk-link">03448 920 908</a>'
```

File: config/locales/en.yml (hunk 13 above)
Comment (huwd) [reply config/locales/en.yml:361]: great spot

File: config/locales/en.yml (hunk 14 above)
Comment (huwd) [reply config/locales/en.yml:401]: great spot

File: config/locales/en.yml [hunk 15]
```diff
@@ -241,3 +241,218 @@ en:
...
+      show_options:
+        - "Yes"
+        - "I might be evicted soon"
+        - "Not sure"
+      links:
+        - text: "Get information about your rights as a tenant"
+          href: "https://www.gov.uk/government/publications/covid-19-and-renting-guidance-for-landlords-tenants-and-local-authorities"
+        - text: "Get coronavirus housing advice from Shelter"
+          href: "https://england.shelter.org.uk/housing_advice/coronavirus"
+        - text: 'If you’re in Northern Ireland call the Housing Executive on <a href="tel:03448 920 908" class="govuk-link">03448 920 908</a>'
```
Comment (injms) [inline config/locales/en.yml:420]:
```suggestion
        - text: 'If you’re in Northern Ireland call the Housing Executive on <a href="tel:+443448920908" class="govuk-link">03448 920 908</a>'
```

File: config/locales/en.yml [hunk 16]
```diff
@@ -241,3 +241,218 @@ en:
...
+          href: "https://www.nidirect.gov.uk/articles/coronavirus-covid-19-taking-care-your-mental-health-and-wellbeing"
+        - text: "Get information about taking care of your mental health if you live in Wales"
+          href: "https://phw.nhs.wales/topics/latest-information-on-novel-coronavirus-covid-19/staying-well-at-home/how-are-you-feeling/"
+        - text: "Get information about coronavirus and your mental wellbeing from the Scottish Association for Mental Health"
+          href: "https://www.samh.org.uk/about-mental-health/self-help-and-wellbeing/coronavirus-and-your-mental-wellbeing"
+        - text: "Find your local loneliness service"
+          href: "https://www.redcross.org.uk/get-help/get-help-with-loneliness/find-your-local-loneliness-service"
+        - text: 'If you need urgent help now, call the Samaritans on <a href="tel:03448 920 908" class="govuk-link">116 123</a>116 123</a>'
+        - text: "If you need urgent help text ‘Shout’ to 85258"
+        - text: 'If you need urgent help and you’re in Northern Ireland call Lifeline on <a href="tel:0808 808 8000" class="govuk-link">0808 808 8000</a>'
```
Comment (injms) [inline config/locales/en.yml:456]:
```suggestion
        - text: 'If you need urgent help and you’re in Northern Ireland call Lifeline on <a href="tel:+448088088000" class="govuk-link">0808 808 8000</a>'
```

File: config/locales/en.yml [hunk 17]
```diff
@@ -241,3 +241,218 @@ en:
...
+        - text: 'If you need urgent help and you’re in Wales call <a href="tel:0800 132 737" class="govuk-link">0800 132 737</a> or text ‘help’ to 81066'
```
Comment (injms) [inline config/locales/en.yml:457]:
```suggestion
        - text: 'If you need urgent help and you’re in Wales call <a href="tel:+44800132737" class="govuk-link">0800 132 737</a> or text ‘help’ to 81066'
```

File: config/locales/en.yml [hunk 18]
```diff
@@ -241,3 +241,218 @@ en:
...
+        - text: 'If it’s an emergency, call <a href="tel:999" class="govuk-link">999</a>'
```
Comment (injms) [inline config/locales/en.yml:458]: Hmm, don't know what to do with the `tel:999` 🤔

Comment (injms) [reply config/locales/en.yml:458]: A quick codepen demo seems to indicate that `<a href="tel:111">Phone 111</a>` works fine on iPhone (didn't really want to phone 999 by accident!) - so I think that this is okay. I'll ask the FE community to see what they think, but this is fine for now.

Comment (huwd) [reply config/locales/en.yml:458]: `tel:+44999`?

Comment (issyl0) [reply config/locales/en.yml:458]: Yeah, I don't feel entirely comfortable with this being a link either - the number of times I mis-hit a link on mobile... 😬

File: config/locales/en.yml [hunk 19]
```diff
@@ -241,3 +241,218 @@ en:
               - "I cannot leave home because I think I’m at risk of severe illness from coronavirus"
               - "I cannot go out because I have a disability"
               - "I’m unable to leave my home for another reason"
+  results_link:
```
Comment (huwd) [inline config/locales/en.yml:244]: Just taken a look at the logic for this page.
So these are just an array of questions right?

We need to do a comparison against the groups /need-help-with choices. Could we scope the questions by group?

Comment (issyl0) [reply config/locales/en.yml:244]: We were thinking something like:

```ruby
if show_options.include?(session[:user_response_to_question])
  <some component-y thing to show each of the links>
end
```

Comment (huwd) [reply config/locales/en.yml:244]: Ah but i think there's a two part show.

We show a group only if they've selected the right section in that checkbox question.

Then... if they've shown a group we can do checks for each question + link

File: config/locales/en.yml (hunk 13 above)
Comment (alex-ju) [reply config/locales/en.yml]: good spot @injms! 🙌

**Pull request merged**